Source/OxyPlot/OxyPlot.Core.nuspec

```diff
@@ -15,10 +15,7 @@
   </metadata>
   <files>
     <file src="..\..\Output\PCL\OxyPlot.???" target="lib\portable-net45+netcore45+wpa81+wp8+MonoAndroid1+MonoTouch1+Xamarin.iOS10" />
-    <file src="..\..\Output\PCL\OxyPlot.???" target="lib\net45" />
-    <file src="..\..\Output\NET40\OxyPlot.???" target="lib\net40-client" />
-    <file src="..\..\Output\NET40\OxyPlot.???" target="lib\net40" />
-
+    <file src="..\..\Output\PCL\OxyPlot.???" target="lib\net45" />    
```

Comment:
I think we can just leave it unchanged for now - it will be fixed when moving to netstandard #946